src/test/java/Homework19.java

```diff
@@ -18,21 +18,22 @@ public void deletePlaylist() throws InterruptedException {
         Assert.assertTrue(avatarIcon.isDisplayed());
 
         //Navigate to playlist (Work Jams)
-        WebElement selectedPlaylist = driver.findElement(By.cssSelector("a[href='#!/playlist/22073']"));
+        WebElement selectedPlaylist = driver.findElement(By.cssSelector("a[href='#!/playlist/35606']"));
         Actions acts = new Actions(driver);
         acts.contextClick(selectedPlaylist).perform();
         Thread.sleep(5000);
 
         //Click "Delete" option
-        WebElement deleteThePlaylist = driver.findElement(By.cssSelector("[data-testid='playlist-context-menu-delete-22073']"));
+        WebElement deleteThePlaylist = driver.findElement(By.cssSelector("[data-testid='playlist-context-menu-delete-35606']"));
         deleteThePlaylist.click();
 
         //Click "OK" confirmation
-        WebElement deleteConfirmation = driver.findElement(By.cssSelector("button[class='ok']"));
-        deleteConfirmation.click();
+//        WebElement deleteConfirmation = driver.findElement(By.cssSelector("button[class='ok']"));
+//        deleteConfirmation.click();
+
         Thread.sleep(3000);
-        String homepage = "https://bbb.testpro.io/#!/home";
-        Assert.assertEquals(driver.getCurrentUrl(), url);
+        String homepageURL = "https://bbb.testpro.io/#!/home";
+        Assert.assertEquals(driver.getCurrentUrl(), homepageURL);
```

Comment:
Please,validate that the playlist is deleted.

```
        WebElement successMessage = driver.findElement(By.cssSelector("div.success"));
        Assert.assertTrue(successMessage.isDisplayed());
```


![image](https://user-images.githubusercontent.com/104761482/211189192-08b0878b-fd1d-4a06-84b2-ce8e5819ea93.png)

Reply:
Thanks for checking the hw. That was my original validation but I was having trouble finding the locator because the message box disappears. I just validated the delete when it automatically goes back to the homepage.  Would you happen to know the css or xpath for the delete confirmation box? Let me know. Thanks!